### ISSUE TITLE
Uptime and Load Averages stop Updating

### ISSUE THREAD

**Issue by HameedSilvertoe** [hexparrot/mineos-node#154]
Hi, I have always had the problem when I log in to the User Interface where it does not display all my servers.  I used to be able to select Refresh Server List and all my servers would then show.  However over the last couple of updates (either 48c7387eb826982b20c63085195c2636ef8e845f or 7834132fe08c4feb0720ed98d4cda721a9711063) when I select to refresh the servers all the servers show however the page freezes (uptime and load averages stop updating).  I'm guessing it's a problem on the client side.  I'm using IE 11.  Should I try a different browser?  Is there one that is recommended?
Thanks...

**Comment by hexparrot** [comment]
I definitely don't recommend using IE11...Internet explorer actually gets zero testing on my end (I test FF, Chrome, and Safari).

Question, does everything break only when you manually refresh the servers?  Is there any consistency between what subset of servers shows up when you log in?

Does `/var/log/mineos.log` show any issues?

Go ahead and try a different browser, but if this issue persists, then let me know the rest of these questions' answers.

**Comment by HameedSilvertoe** [comment]
I'd rather not have to download and install a web browser for a single use like this if I don't have to but will. It may take me a day or so to get another browser downloaded and installed as I have a fairly slow connection.

The only server that ever appears upon first log in is the first one in the list.  I assume that the servers are always listed alphabetically therefore I guess that something is happening at or after the first server is listed?  When I refresh I can still use some of the links but not all, for instance I can click and go to the server status page, however, I cannot change the profile or runnable jar.  I was able to do both on the first server before I refreshed the server list.

It does look like the log is showing issues.
Thanks
[mineos.docx](https://github.com/hexparrot/mineos-node/files/59064/mineos.docx)

**Comment by HameedSilvertoe** [comment]
I cleared the log, installed and logged in using FF and everything worked much smoother.  The log now only has info messages so I was correct in my triage that the problem was with IE.  I have seen the issue where not all the servers will show on the initial login on FF once already but it was only one server that did not show with FF instead of only one server showing in IE.  When I refreshed the list in FF it continued to run normally without freezing so FF has either fixed the issue or is a good workaround.  Thanks for your efforts on MineOs.  I really enjoy using it.
